### ISSUE TITLE
[meta] Ask GitHub to mark 'gccrs' as a mirror of 'gcc'

### ISSUE THREAD

**Issue by tschwinge** [Rust-GCC/gccrs#323]
@philberty, may I ask GitHub to mark 'gccrs' as a mirror of 'gcc', <https://github.com/gcc-mirror/gcc>?  This should make some things more natural in peoples' workflow: can use existing GCC fork for submitting GCC/Rust pull requests, for example.

**Comment by philberty** [comment]
Yes, go for it that sounds really interesting I didn't know we could do that :)

**Comment by dkm** [comment]
Have you seen this ? https://twitter.com/gnutools/status/1354103633567211520 :)

**Comment by tschwinge** [comment]
> Have you seen this ? https://twitter.com/gnutools/status/1354103633567211520 :)

Yes, but that's a separate "issue" (not sure about its status), and shouldn't impede this one -- or do you think differently?

**Comment by tschwinge** [comment]
I've filed <https://support.github.com/ticket/1089378> (only visible to me, I suppose).

**Comment by dkm** [comment]
No no I don't think it's an issue, I was simply surprised by this info (and the lack of followup)
